server/controller/src/main/java/ai/starwhale/mlops/exception/SwValidationException.java

```diff
@@ -71,7 +71,8 @@ public enum ValidSubject {
         ONLINE_EVAL("015", "Online Eval"),
         UPGRADE("016", "Upgrade"),
         TAG("017", "Resource Tag"),
-        REPORT("018", "Report");
+        REPORT("018", "Report"),
+        FINE_TUNE("019", "Fine Tune");
```

Comment:
Is this thicker? I think the previous features were at the operational level. This is at the conceptual level.